zeroconf/test.py

```diff
@@ -16,8 +17,11 @@
 
 from nose.plugins.attrib import attr
 
-import zeroconf as r
-from zeroconf import (
+# ensure I can find this package even when it hasn't been installed (for development purposes)
+sys.path.insert(0, '..')
```

Comment:
This is better done by setting `PYTHONPATH` when running tests/importing. Manipulating `sys.path` (particularly at import time) is a dangerous affair and let's avoid doing it.

Reply:
Thanks for the explanation, will do.